packages/gatsby-link/src/__tests__/index.js

```diff
@@ -19,6 +19,12 @@ const getNavigateTo = () => {
 }
 
 describe(`<Link />`, () => {
+
+  it(`does not fail to initialize when __PREFIX_PATHS__ is not defined`, () => {
```

Comment:
> As for the eslint comments

Only because (as far as I remember), I haven't seen any in the existing code-base, and it's pretty typically advisable to kinda keep that stuff as-is unless for a good reason! However I'll defer to @KyleAMathews on that one!